src/features/user/Settings/SwitchUser.tsx

```diff
@@ -0,0 +1,80 @@
+import { useTranslation } from 'next-i18next';
+import styles from './SwitchUser.module.scss';
+import { SwitchUserContainer } from '../../common/Layout/SwitchUserContainer';
+import { TextField, Button } from '@mui/material';
+import { useForm } from 'react-hook-form';
+import { ParamsContext } from '../../common/Layout/QueryParamsContext';
+import { useContext } from 'react';
+
+const SwitchUser = () => {
+  const { t } = useTranslation('me');
+
+  const { register, errors, handleSubmit } = useForm({
+    mode: 'onSubmit',
+  });
+  const { targetEmail, setTargetEmail, setAlertError, alertError, setEmail } =
+    useContext(ParamsContext);
+
+  const handle = () => {
+    setAlertError(false);
+    if (targetEmail) {
+      setEmail(targetEmail);
+      setTargetEmail('');
+    }
+  };
+
+  return (
+    <>
```

Comment:
@sunilsabatp 
1. When possible, we should reuse existing components. Refer to the way src/features/user/Widget/DonationLink/index.tsx is structured.
2. Avoid introducing new global css into the project.